src/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -15,45 +15,49 @@
  * See the License for the specific language governing permissions and 
  * limitations under the License. 
  */
-package org.zaproxy.zap.extension.httpsInfo;
+package org.zaproxy.zap.extension.httpsinfo;
```

Comment:
That does work. Sadly v6 doesn't allow dynamic unloading so a reboot is required, but it does all seem to behave as wanted in the end.